domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -434,15 +490,22 @@ public static void closeAllClients() {
     }
     PG_POOLS.values().forEach(PgPool::close);
     PG_POOLS.clear();
+    PG_POOLS_READER.values().forEach(PgPool::close);
+    PG_POOLS_READER.clear();
   }
 
-  static PgConnectOptions createPgConnectOptions(JsonObject sqlConfig) {
+  static PgConnectOptions createPgConnectOptions(JsonObject sqlConfig, String desiredHost) {
```

Comment:
Replacing `String desiredHost` by `boolean isReader` might be easier to understand and implement.